skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -7,6 +7,11 @@
 #include "ServerState.h"
 #include "WorldState.h"
 #include <NiPoint3.h>
+#include <random>
+#include <string>
+
+std::random_device rd;
+std::mt19937 gen(rd());
```

Comment:
let's create `rd` and `gen` each time they are needed. not globally. seems to be unsafe due to statics initialization order

Reply:
`std::mt19937 g_rng{std::random_device{}()};` should be safe I think

Reply:
@nic11 I like this